templates/jobs/crypt-vault-bootstrap.tmpl.yaml

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
This template is applied after the `start.sh` script unseals the vault instance. It will create vault policies and mounts based on the rules provided in `bootstrap-rules.json`, and write the output tokens to the `crypt-tokens` and `gateway-tokens` secrets for inclusion as environment variables in other services.

The `bootstrap_docker_image` has been open-sourced although the repo itself is currently still private. We will open-source the repo as well shortly, although the same functionality could also be inlined into this repo.